ui/src/main/java/com/fincity/saas/ui/document/URIPath.java

```diff
@@ -0,0 +1,140 @@
+		this.whitelist = CloneUtil.cloneMapList(uriPath.whitelist);
+		this.blacklist = CloneUtil.cloneMapList(uriPath.blacklist);
+		this.kiRunFxDefinition = CloneUtil.cloneObject(uriPath.kiRunFxDefinition);
+	}
+
+	@SuppressWarnings("unchecked")
+	@Override
+	public Mono<URIPath> applyOverride(URIPath base) {
+		if (base == null) {
+			return Mono.just(this);
+		}
+
+		return FlatMapUtil.flatMapMonoWithNull(
+				() -> DifferenceApplicator.apply(this.httpMethods, base.httpMethods),
+				hm -> DifferenceApplicator.apply(this.headers, base.headers),
+				(hm, h) -> DifferenceApplicator.apply(this.queryParams, base.queryParams),
+				(hm, h, qp) -> DifferenceApplicator.apply(this.whitelist, base.whitelist),
+				(hm, h, qp, w) -> DifferenceApplicator.apply(this.blacklist, base.blacklist),
+				(hm, h, qp, w, b) -> {
+					this.httpMethods = (List<HttpMethod>) hm;
+					this.headers = (List<String>) h;
+					this.queryParams = (List<String>) qp;
+					this.whitelist = (List<String>) w;
+					this.blacklist = (List<String>) b;
+
+					if (this.pathString == null)
+						this.pathString = base.pathString;
+					if (this.uriType == null)
+						this.uriType = base.uriType;
+					if (this.isAuthenticated == null)
+						this.isAuthenticated = base.isAuthenticated;
+					if (this.kiRunFxDefinition == null)
+						this.kiRunFxDefinition = CloneUtil.cloneObject(base.kiRunFxDefinition);
+					if (this.redirectionDefinition == null)
+						this.redirectionDefinition = CloneUtil.cloneObject(base.redirectionDefinition);
+
+					return Mono.just(this);
+				}).contextWrite(Context.of(LogUtil.METHOD_NAME, "URIPath.applyOverride"));
+	}
+
+	@SuppressWarnings("unchecked")
+	@Override
+	public Mono<URIPath> makeOverride(URIPath base) {
+		if (base == null) {
+			return Mono.just(this);
+		}
+
+		return FlatMapUtil.flatMapMonoWithNull(
+				() -> Mono.just(this),
+				obj -> DifferenceExtractor.extract(obj.httpMethods, base.httpMethods),
+				(obj, hm) -> DifferenceExtractor.extract(obj.headers, base.headers),
+				(obj, hm, h) -> DifferenceExtractor.extract(obj.queryParams, base.queryParams),
+				(obj, hm, h, qp) -> DifferenceExtractor.extract(obj.whitelist, base.whitelist),
+				(obj, hm, h, qp, w) -> DifferenceExtractor.extract(obj.blacklist, base.blacklist),
+				(obj, hm, h, qp, w, b) -> {
+					obj.setHttpMethods((List<HttpMethod>) hm);
+					obj.setHeaders((List<String>) h);
+					obj.setQueryParams((List<String>) qp);
+					obj.setWhitelist((List<String>) w);
+					obj.setBlacklist((List<String>) b);
+
+					if (obj.pathString != null && obj.pathString.equals(base.pathString))
+						obj.pathString = null;
+					if (obj.uriType != null && obj.uriType.equals(base.uriType))
+						obj.uriType = null;
+					if (obj.isAuthenticated != null && obj.isAuthenticated.equals(base.isAuthenticated))
+						obj.isAuthenticated = null;
+					if (obj.kiRunFxDefinition != null && obj.kiRunFxDefinition.equals(base.kiRunFxDefinition))
```

Comment:
Need to use Difference Extractor for both kirundef and redirect differentiator extraction and appllication.